core/src/main/java/discord4j/core/spec/MessageCreateSpec.java

```diff
@@ -45,10 +45,9 @@
  *
  * @see <a href="https://discord.com/developers/docs/resources/channel#create-message">Create Message</a>
  */
-public class MessageCreateSpec implements Spec<MultipartRequest> {
+public class MessageCreateSpec implements Spec<MultipartRequest>, Appendable {
```

Comment:
Not sure if implementing `Appendable` is a good idea as its method signatures throw IOExceptions

Reply:
Is methods throwing IOException's bad practice?

Reply:
A) We're not doing any IO whatsoever so it's misleading at best.
B) It's a checked exception. Working with checked exceptions and lambdas (which you do a lot with this library) is not a good experience.